bc_obps/compliance/service/payment_instructions_service.py

```diff
@@ -16,19 +16,19 @@ class PaymentInstructionsService:
     @classmethod
     def generate_payment_instructions_pdf(
         cls,
-        compliance_summary_id: int,
+        compliance_report_version_id: int,
     ) -> Tuple[Generator[bytes, None, None], str, int]:
         """
         Generate a PDF payment instructions and return a generator that yields chunks of the PDF data.
 
         Args:
-            compliance_summary_id: ID of the compliance summary
+            compliance_report_version_id: ID of the compliance summary
 
         Returns:
             Tuple of (PDF data generator, filename, total_size_in_bytes)
         """
         try:
-            context = PaymentInstructionsService._prepare_payment_instructions_context(compliance_summary_id)
```

Comment:
I think this must be the report version id because that's the name of the arg in `_prepare_payment_instructions_context`